### PR TITLE
Bugfix/#3012 sonar bugs fix

### PR DESCRIPTION
**Pull request by dyadyushko**
refactor code accordingly to sonar remarks, in order to reduce code complexity and upgrade readability.

### CODE REVIEW

**Review by rostikdoroshenko — APPROVED** [review]

**Review by YuriShevchuk — APPROVED** [review]

**Review by s1auka — APPROVED** [review]

File: src/app/main/component/admin/components/feedbacks/feedbacks.component.ts [hunk 1]
```diff
@@ -76,7 +76,9 @@ export class FeedbacksComponent implements OnInit {
     );
   }
 
-  showMessage(message: string, messageType: string) { }
+  showMessage(message: string, messageType: string) {
+    // TODO: add functionality to this method
+  }
```
Comment (JimHolman1199) [inline src/app/main/component/admin/components/feedbacks/feedbacks.component.ts:81]: can we just remove an empty method?

File: src/app/main/component/admin/components/old-places/old-places.component.ts [hunk 2]
```diff
@@ -205,7 +205,9 @@ export class OldPlacesComponent implements OnInit {
     }
   }
 
-  showMessage(message: string, messageType: string) { }
+  showMessage(message: string, messageType: string) {
+    // TODO: add functionality to this method
+  }
```
Comment (JimHolman1199) [inline src/app/main/component/admin/components/old-places/old-places.component.ts:210]: can we remove an empty method ?

File: src/app/main/component/admin/components/users/users.component.ts [hunk 3]
```diff
@@ -100,9 +100,13 @@ export class UsersComponent implements OnInit {
...
 
-  successfulAction(message: string) { }
+  successfulAction(message: string) {
+    // TODO: add functionality to this method
+  }
 
-  errorMessage(message: string) { }
+  errorMessage(message: string) {
+    // TODO: add functionality to this method
+  }
```
Comment (JimHolman1199) [inline src/app/main/component/admin/components/users/users.component.ts:109]: the same

File: src/app/main/component/shared/components/form-base/form-base.component.ts [hunk 4]
```diff
@@ -29,7 +29,9 @@ export class FormBaseComponent implements ComponentCanDeactivate {
     }
   };
 
-  public getFormValues(): any {}
+  public getFormValues(): any {
+    // TODO: add functionality to this method
+  }
```
Comment (JimHolman1199) [inline src/app/main/component/shared/components/form-base/form-base.component.ts:34]: the same

File: src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts [hunk 5]
```diff
@@ -36,17 +35,25 @@ export class SocialNetworksComponent implements ControlValueAccessor {
...
 
-  onChange = (value: any) => {};
+  onChange(value: any) {
+    // TODO: add functionality to this method
+  }
 
-  writeValue(obj: any): void {}
+  writeValue(obj: any): void {
+    // TODO: add functionality to this method
+  }
```
Comment (JimHolman1199) [inline src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts:44]: the same

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts [hunk 6]
```diff
@@ -91,13 +91,6 @@ export class UBSOrderDetailsComponent extends FormBaseComponent implements OnIni
     return this.showTotal > 0;
   }
 
-  translateWords(key: string, variable) {
-    return this.translate
-      .get(key)
-      .pipe(take(1))
-      .subscribe((item) => (variable = item));
-  }
```
Comment (JimHolman1199) [inline src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts:99]: Remove unused import of 'take' from this file

File: src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts (hunk 5 above)
Comment (dyadyushko) [reply src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts:44]: Are you sure about removing these methods? Though they are empty, their existence are notification that something not quite done.

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts (hunk 6 above)
Comment (dyadyushko) [reply src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts:99]: done

File: src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts (hunk 5 above)
Comment (JimHolman1199) [reply src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts:44]: yes, remove them

Comment (dyadyushko) [reply src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts:44]: done

**Pull request merged**